Documentation/operations/troubleshooting.rst

```diff
@@ -537,6 +537,93 @@ Opening the ``status``, we can drill down through ``policy.realized.l4``. Do
 your ``ingress`` and ``egress`` rules match what you expect? If not, the
 reference to the errant rules can be found in the ``derived-from-rules`` node.
 
+Policymap pressure and overflow
+-------------------------------
+
+The most important step is finding out which node is experiencing policymap
+pressure.
+
+The ``cilium_bpf_map_pressure{map_name="cilium_policy_*"}`` metric monitors the
+endpoint's BPF policymap pressure. This metric exposes the maximum BPF map
+pressure on the node, meaning the policymap experiencing the most pressure on a
+particular node.
+
+Once the node is known, the troubleshooting steps are as follows:
+
+1. Find the Cilium pod on the node experiencing the problematic policymap
+   pressure and obtain a shell via ``kubectl exec``.
+2. Use ``cilium policy selectors`` to get an overview of which selectors are
+   selecting many identities. The output of this command as of Cilium v1.15
+   additionally displays the namespace and name of the policy resource of each
+   selector.
+3. The type of selector tells us what sort of policy rule could be having an
+   impact. There are 3 types of selectors are explained below, each with an
+   specific steps depending on the selector.
+4. Consider bumping the policymap size as a last resort.
```

Comment:
>as a last resort

What would be the implications of increasing policy map size besides increasing the memory footprint?

Reply:
Added some context here.